packages/freezed/pubspec.yaml

```diff
@@ -14,7 +15,8 @@ dependencies:
   build_config: ^0.4.2
   meta: ^1.3.0
   source_gen: ^0.9.0
-  freezed_annotation: ">=0.14.0 <2.0.0"
+  freezed_annotation:
```

Comment:
I'd suggest using `melos` instead, with the command `melos boostrap`

Reply:
Cool, didn't know about that, will take a look.